deployment/persistent_chains/testnet-2.0/testnet-2.0-launch.md

```diff
@@ -303,3 +303,13 @@ systemctl restart dcld
   * account status:
     * `dcld query auth account --address="<address>"`: to ensure that account is created and has assigned role
 <!-- markdownlint-enable MD029 -->
+
+## VII. Post-Ceremony: Adding new nodes to testnet
+When adding new nodes to testnet after a while, you might consider one of the options described in [running-node-in-existing-network](../../../docs/running-node-in-existing-network.md). <br>
+To configure recommended `state sync` option described in the document you can optionally use the following available `CSA Testnet2.0` endpoints:
+- `REST` - https://on.test-net.dcl.csa-iot.org
```

Comment:
It may make sense to also add these values into a separate file within https://github.com/zigbee-alliance/distributed-compliance-ledger/tree/master/deployment/persistent_chains/testnet-2.0 so that they can be found easier.

Reply:
We can also add simple how-to instructions (commands to be copied) to that new file describing how to add a new ON connected to CSA's ONs.